src/arduino/arduino.ts

```diff
@@ -732,6 +732,11 @@ export class ArduinoApp {
             }
             if (verbose) {
                 arduinoChannel.channel.append(line);
+            } else {
+                // Output sketch memory usage in non-verbose mode
+                if (line.startsWith("Sketch uses ") || line.startsWith("Global variables use ")) {

```

Comment:
I suggest replacing these two tests with a `isMemoryUsageInformation(line)` function call instead.

First, this ought to be localizable since the output from arduino might not be in English, e.g. from https://github.com/arduino/Arduino/blob/master/arduino-core/src/processing/app/i18n/Resources_tr.po:

```
msgid ""
"Sketch uses {0} bytes ({2}%%) of program storage space. Maximum is {1} "
"bytes."
msgstr "Çalışmanız programın {0} bayt ({2} %%) saklama alanını kullandı. Maksimum {1} bayt."
```
Now, only supporting English as a first step in the beginning might be ok, but at least i18n should be on the map and not forgotten.


And secondly it is just good separation of concerns to split out the parsing details to a separate function. What if the output text changes slightly in newer versions of arduino? There is no logic here in this function that should require any change because of that.

Reply:
I agree, the solution is not ideal, this was meant as a hotfix (that may be improved later) in hopes it could resolve the issue for users now.
I will separate the condition into a separate function, as you've suggested.

Reply:
@hlovdal is that alright?

Reply:
The `isMemoryUsageInformation` function looks good. But I have no authority to approve this pull request if that's your question. I have just contributed a few things before.